TheHiveV5/action_add_observable.json

```diff
@@ -0,0 +1,119 @@
+{
+  "uuid": "3b3ab2df-8b4f-49ce-ae69-560722b1b14a",
+  "name": "Add observables in The Hive v5",
+  "description": "Action to add a list of observables in The Hive v5",
+  "docker_parameters": "thehive_add_observable",
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "alert_id": {
+        "description": "The id of the alert you want to add observables to",
+        "default": "~4231176",
```

Comment:
It's a default value or an example? If it's an example, can you remove this field?

```suggestion
```